### DISCUSSION TITLE
🖊️ Better error message for 'backward'

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#5995] [Language Ideas]
At the moment Hedy gives the 'you forgot to use a command' error when using backward in the turtle. While I think that would be a very logical command for a student to try. So maybe create an error message that tells you backward isn't a command but you can use forward and negative numbers. 

![image](https://user-images.githubusercontent.com/80678586/153416983-80623aae-80e7-40bb-a31f-a96536cde7fc.png)

**Comment by huang-v** [comment]
Would a custom error message for the first level be desired?

**Comment by MarleenGilsing** [comment]
@huang-v Yes it is, thanks! I would like the error message to make clear that the turtle can go backwards but they have to use forward -100.

**Comment by Felienne** [comment]
Yeah we have this nice string distance error for other keywords, maybe we want to use that here too?
